app/services/uploader.js

```diff
@@ -1,12 +1,36 @@
 'use strict';
 const angular = require('angular');
 const app = angular.module('app');
+const urlParse = require('url').parse;
 
 app.factory('organizerUpload', organizerUpload);
 
 organizerUpload.$inject = ['apiQueues'];
 
+const localHostnames = new Set([
+  'docker.local.flywheel.io',
+  'localhost'
+]);
+
 function organizerUpload(apiQueues) {
+  function _request(options) {
+    let url = options.url;
+    if (url[0] === '/' && options.instance) {
+      // this removes port when instance is something like localhost:8080
+      const hostname = urlParse('test://' + options.instance).hostname;
+      const scheme = localHostnames.has(hostname) ? 'http' : 'https';
```

Comment:
I think https://github.com/scitran/organizer/pull/67/commits/d3a50b8061ca3ffb0f25ae1913995009306955bd will help with this issue. `npm run dev-insecure` should solve the issue.